kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/Methods.java

```diff
@@ -6,15 +6,27 @@
 //- @Methods defines/binding Class
 public class Methods {
 
+  //- ArrayBuiltin=vname("array#builtin","","","","java").node/kind tbuiltin
+  //- FnBuiltin=vname("fn#builtin","","","","java").node/kind tbuiltin
+  //- IntBuiltin=vname("int#builtin","","","","java").node/kind tbuiltin
+  //- VoidBuiltin=vname("void#builtin","","","","java").node/kind tbuiltin
+
   //- @Methods defines/binding Ctor
   //- Ctor.node/kind function
   //- Ctor childof Class
   //- Ctor typed CtorType
   //- CtorType param.0 FnBuiltin
   //- CtorType param.1 Class
+  //- CtorType param.2 Class
```

Comment:
I'm not sure this is entirely right, but I don't have any alternatives I like better.

Reply:
Also, it looks like the schema document needs updated.